variables.tf

```diff
@@ -565,3 +565,20 @@ variable "aws_gateway_api_controller" {
   type        = any
   default     = {}
 }
+
+################################################################################
+# GitOps Bridge
+################################################################################
+
+# Only required when enable_aws_gateway_api_controller is true
+variable "vpc_id" {
```

Comment:
@bryantbiggs what do you think about this new variable?
It makes a cleaner interface for the gitops bridge metadata.

It allows something like this
```hcl
module "gitops_bridge_metadata" {
  source = "../../../modules/gitops-bridge-metadata"

  cluster_name = module.eks.cluster_name
  metadata = module.eks_blueprints_addons.gitops_metadata
  environment = local.environment
  addons = local.addons
}
```
If `metadata_aws_vpc_id` doesn't come from `module.eks_blueprints_addons.gitops_metadata`
then it looks like this
```hcl
module "gitops_bridge_metadata" {
  source = "../../../modules/gitops-bridge-metadata"

  cluster_name = module.eks.cluster_name
  metadata = merge(module.eks_blueprints_addons.gitops_metadata, {
     metadata_aws_vpc_id = module.vpc.vpc_id
  })
  environment = local.environment
  addons = local.addons
}
```
I'm ok either way, we only need to doc if doing api gateway controller to pass the extra vpcid, if we leave it we could pass the vpc id into the helm_release when installing helm chart using terraform provider

Reply:
apologies, this one slipped off my radar. In short - we should avoid passing through values and only export values created by this module. I'll take a deeper look at this PR tomorrow (8/9)

Reply:
I remove the variable vpc_id and updated the examples to pass it to the metadata module